src/@next/Tooltip/Tooltip.tsx

```diff
@@ -147,7 +147,7 @@ export type TooltipProps = {
   zIndex?: number;
   clickable?: boolean; // if true, tooltip will be shown on click instead of hover
   timeout?: number; // how long tooltip will still be shown after clicked or when not hovered anymore
-  onClick?: () => void; // onClick will be called when tooltip is clicked or hovered
+  onAction?: () => void; // onAction will be called when tooltip is clicked or hovered out
```

Comment:
I just realised, I forgot to mention this in previous PR but can you put the comments in this format:
```ts
 /** description of prop here */
prop: propType;
```

so it'll appear as description in the storybook